frontend/src/store/shoot/helper.js

```diff
@@ -414,6 +414,8 @@ export function getRawVal (context, item, column) {
       return getIssueSince(item.status) || 0
     case 'technicalId':
       return item.status?.technicalID
+    case 'workers':
+      return item.spec.provider.workers?.length || 'workerless'
```

Comment:
I all other places we have `item?.spec?.provider?.workers?.length` The possible results of this expression are `['workerless', 1, 2, 3, ...]`. Why not `[0, 1, 2, 3, ...]`?

Reply:
because this value is used not only to sort but also to search / filter the table. The worker column shows the worker groups component which displays `Workerless Cluster`. It shall be possible to easily filter for these clusters, this is why I return "workerless" in this case